storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -128,3 +178,29 @@ export const CheckboxGroup: Story = {
     </FieldSet>
   ),
 }
+
+// export const CheckboxGroupWithError: Story = {
+//   args: {
+//     invalid: true,
+//     legend: 'Waar gaat uw melding over?',
+//   },
+//   render: (args) => (
+//     <FieldSet legend={args.legend} invalid={args.invalid}>
+//       {args.invalid && <ErrorMessage className="ams-mb--sm">Geef aan waar uw melding over gaat.</ErrorMessage>}
+//       <Column gap="extra-small" style={{ width: '100%' }}>
+//         <Checkbox name="about" value="horeca" invalid={args.invalid} aria-required="true">
+//           Horecabedrijf
+//         </Checkbox>
+//         <Checkbox name="about" value="ander_bedrijf" invalid={args.invalid} aria-required="true">
+//           Ander soort bedrijf
+//         </Checkbox>
+//         <Checkbox name="about" value="evenement" invalid={args.invalid} aria-required="true">
+//           Evenement
+//         </Checkbox>
+//         <Checkbox name="about" value="anders" invalid={args.invalid} aria-required="true">
+//           Iets anders
+//         </Checkbox>
+//       </Column>
+//     </FieldSet>
+//   ),
+// }
```

Comment:
Why is there commented-out code here? WIP?

Reply:
I tried to explain that [here](https://github.com/Amsterdam/design-system/pull/1247/files#diff-69cbe174562903ae5ab20b810f196ff8bd28d15cbf97a08f9a91781c6d5db1d8R63). There is a bug with NVDA and fieldsets that use `aria-described`. The commented out story works for other screen readers, but we have to find a workaround for the NVDA bug before we can show this story (imo).